examples/puppetmaster/docker-compose.yml

```diff
@@ -1,10 +1,10 @@
-version: "2"
+version: "3"
 
 services:
   puppet:
     container_name: puppet
     hostname: puppet
-    image: puppet/puppetserver
+    image: puppet/puppetserver:5.3.7
```

Comment:
was this failing with the latest? is this the highest 5.x?

Reply:
Correct - `latest` is `6.x.y` and this test does not work with that version so this is the highest `5.x.y` that passes. They also don't have `5` nor `5.x` aliases so you have to specify the exact version.